src/Control/Bind.purs

```diff
@@ -63,6 +63,17 @@ infixr 1 bindFlipped as =<<
 instance bindFn :: Bind ((->) r) where
   bind m f x = f (m x) x
 
+-- | The `Array` monad works like a nested for loop. Each
+-- | `bind`/`>>=` adds another level of nesting in the loop:
+-- | ```
+-- | foo :: Array Int
+-- | foo = do
```

Comment:
Best to remove this `do` with this change, I think

Reply:
Done